### PR TITLE
feat: replace c-log by riverr

### PR DESCRIPTION
**Pull request by WillVill**
---
name: Add Riverr as collaborator
about: Add Riverr as collaborator and remove C-LOG
---

### CODE REVIEW

**Comment by WillVill** [comment]
Sorry for the messy PR

**Comment by WillVill** [comment]
Just wanted to illustrate via the PR that I am the same person who added C-LOG as a collaborator and removing it.

File: src/components/Collaborate/collaborators.json [hunk 1]
```diff
@@ -8,12 +8,11 @@
     "logo": "/static/images/collaborate/CONVERGENCE_logo.png"
   },
   {
-    "id": "clog",
-    "name": "C-LOG",
-    "description": "Provides a trusted universal industry standard for ID and credentials through a blockchain-enabled decentralised identifier.",
-    "website": "https://c-log.io",
-    "key": "clog",
-    "logo": "/static/images/collaborate/CLOG_logo.png"
```
Comment (Nebulis) [inline src/components/Collaborate/collaborators.json:16]: please remove the old logo

File: src/components/Collaborate/collaborators.json [hunk 2]
```diff
@@ -8,12 +8,11 @@
...
+    "name": "Riverr",
```
Comment (Nebulis) [inline src/components/Collaborate/collaborators.json:11]: add `id: riverr`

**Review by Nebulis — APPROVED** [review]

**Pull request merged**